mongo/datadog_checks/mongo/collectors/base.py

```diff
@@ -107,6 +107,17 @@ def _submit_payload(self, payload, additional_tags=None, metrics_to_collect=None
                 if isinstance(metrics_to_collect[metric_name], tuple)
                 else metric_name
             )
+
+            # This is because https://datadoghq.atlassian.net/browse/AGENT-9001
+            # Delete this code when the metrics are definitely deprecated
+            if (
+                metric_name_alias == 'opLatencies.reads.latency'
```

Comment:
Consider also: `if metric_name_alias in ('opLatencies.reads.latency', 'opLatencies.writes.latency', 'opLatencies.commands.latency'):

Reply:
Good point. Done